python/lsst/sdm/schemas/apdb.yaml

```diff
@@ -1967,3 +1227,1110 @@ tables:
+
+  - name: fk_SSSource_mpc_orbits
+    description: Link an SSSource to its associated mpc_orbits
+    '@type': ForeignKey
+    columns:
+    - '#SSSource.designation'
+    referencedColumns:
+    - '#mpc_orbits.designation'
+
+  indexes:
+  - name: idx_SSSource_diaSourceId
+    description: >
+      Unique index on the diaSourceId column; accelerates joins between
+      DiaSource and SSSource.
+    columns:
+    - "#SSSource.diaSourceId"
+
+  - name: idx_SSSource_ssObjectId
+    description: >
+      Non-unique index on the ssObjectId column; accelerates retrieval of
+      single-epoch data for SSObjects.
+    columns:
+    - "#SSSource.ssObjectId"
+
+  - name: idx_SSSource_designation
+    description: >
+      Non-unique index on the designation column; accelerates retrieval of
+      single-epoch data for SSObjects.
+    columns:
+    - "#SSSource.designation"
+
+- name: mpc_orbits
+  description: Table of orbital elements and related information of known (sun-orbiting and unbound) Solar
+    System objects. Replicated from the Minor Planet Center (Postgres) database. This schema will
+    generally closely follow the schema of the upstream table, to allow end-users to rerun queries
+    developed elsewhere on the RSP.
+  tap:table_index: 130
+  primaryKey: '#mpc_orbits.id'
+  columns:
+  - name: id
+    description: Internal ID (generally not seen/used by the user)
+    datatype: int
+    nullable: false
+    ivoa:ucd: meta.id;src
+  - name: designation
+    # Note: the 'designation' column is a Rubin-provided copy of the
+    # unpacked_primary_provisional_designation column to work around UX issues.
+    description: The primary provisional designation in unpacked form (e.g. 2008 AB).
+    datatype: char
+    length: 16
+    nullable: false
+    ivoa:ucd: meta.id;src
+  - name: packed_primary_provisional_designation
+    description: The primary provisional designation in packed form (e.g. K08A00B)
+    datatype: char
+    length: 16
+    nullable: false
+    ivoa:ucd: meta.id;src
+  - name: unpacked_primary_provisional_designation
+    description: The primary provisional designation in unpacked form (e.g. 2008 AB)
+    datatype: char
+    length: 16
+    nullable: false
+    ivoa:ucd: meta.id;src
+  - name: mpc_orb_jsonb
+    description: Details of the orbit solution in JSON form
+    datatype: text
+  - name: created_at
+    description: When this row was created
+    datatype: timestamp
+    precision: 6
+    ivoa:ucd: time.creation;meta.dataset
+  - name: updated_at
+    description: When this row was updated
+    datatype: timestamp
+    precision: 6
+    ivoa:ucd: time.processing;meta.dataset
+  - name: orbit_type_int
+    description: Orbit Type (Integer)
+    datatype: int
+  - name: u_param
+    description: U parameter
+    datatype: int
+  - name: nopp
+    description: number of oppositions
+    datatype: int
+  - name: arc_length_total
+    description: "Arc length over total observations [days]"
+    datatype: double
```

Comment:
Do we need double precision for all columns below?

Reply:
We replicate these tables from the MPC, and keep the exact schema (and docs) as upstream.